static/js/tools/scatter2/chart.tsx

```diff
@@ -134,11 +162,19 @@ function getYStd(points: Array<Point>): string {
  * @param props
  */
 function plot(
-  svg: React.MutableRefObject<SVGElement>,
+  svgRef: React.MutableRefObject<SVGElement>,
+  svgContainerRef: React.MutableRefObject<HTMLDivElement>,
   tooltip: React.MutableRefObject<HTMLDivElement>,
   props: ChartPropsType
 ): void {
-  d3.select(svg.current).selectAll("*").remove();
+  d3.select(svgRef.current).selectAll("*").remove();
+  const svgContainerWidth = Math.max(svgContainerRef.current.offsetWidth, 400);
+  const svgContainerHeight = Math.max(300, svgContainerWidth / 3);
```

Comment:
ideally we'd convert the plot container to a flexbox, with 2 fixed panes (the search and plot options), with the chart expanding to fill the remaining space. this is good for now tho